src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Resources/xlf/Strings.cs.xlf

```diff
@@ -27,7 +27,7 @@
         <target state="translated">Typ elementu kolekce se nepodporuje: {0}“.</target>
         <note />
       </trans-unit>
-      <trans-unit id="LanguageVersionIsNotSupportedMessageFormat">
+      <trans-unit id="Language VersionIsNotSupportedMessageFormat">
```

Comment:
this looks like an error, we shouldn't have a space in the ID

Reply:
Per the PR description, it looks like they want us to open a bug at https://aka.ms/icxLocBug.  I'll give that a try and share what I learn so that folks can do that in the future.